Gems/Atom/Asset/Shader/Code/Source/Editor/ShaderVariantAssetBuilder.cpp

```diff
@@ -62,6 +62,7 @@ namespace AZ
     namespace ShaderBuilder
     {
         static constexpr char ShaderVariantAssetBuilderName[] = "ShaderVariantAssetBuilder";
+        static constexpr char RgaExePath[] = "/_deps/rga-src/rga.exe";
```

Comment:
always windows for now I suppose?

Reply:
I added Linux support, but I have no machine to test